app/models/data.js

```diff
@@ -52,4 +52,26 @@ Data.prototype.delete = function (key, callback) {
   });
 };
 
+Data.prototype.deleteOlderThanDate = function (date, callback) {
+  this.db.find({}, (err, docs) => {
+    if (err) {
+      return callback(err);
+    }
+
+    if (docs.length > 0) {
+      docs.forEach((item)=>{
+        let updatedAtDate = new Date(item.updatedAt);
+        if (updatedAtDate.getTime() < date) {
```

Comment:
@tejohnso base on the data that is on the data base is format like this:
```
{"key":"risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_","value":"","_id":"8MBqZbqNaf9nHi8S","createdAt":{"$$date":1501610701895},"updatedAt":{"$$date":1501524301895}}
```

I tried to filter directly using the followed two ways. First trying to delete directly filtering and trying to find filtering. None of them worked so I had to go dirty and retrieve all then filter and call to delete. Performance wise it was pretty good. It cleaned up a 200MB file in seconds. So I don't think it going to be a problem. 

```
Data.prototype.deleteOlderThanDate = function (date, callback) {
  this.db.remove({ updatedAt: { $lt: date } }, (err, numRemoved) => {
    if (err) {
      return callback(err);
    }

    callback(null, numRemoved);
  });
};
```

```
  this.db.find({ "updatedAt.$$date": { $lt: date } }, {}, (err, docs) => {
    if (err) {
      return callback(err);
    }

    callback(null, docs);
  });
```